.github/workflows/ci_cd.yml

```diff
@@ -519,6 +520,56 @@ jobs:
       - name: Pass
         run: echo 'Django checks are skipped because API is unchanged.'
 
+  py-package-checks:
+    name: Run checks for Python packages/*
+    if: |
+      needs.get-changes.outputs.py_packages == 'true'
+    runs-on: ubuntu-latest
+    needs:
+      - get-changes
+      - lint
+    strategy:
+      fail-fast: false
+      matrix:
+        packages:
+          - packages/python/openverse-attribution
```

Comment:
It would be nice to be able to have this directory listed automatically in a previous step, and then use it to build the `packages` matrix, instead of hard-coding it. This applies to the pnpm packages too. What do you think? Shall I open an issue for it?